Pipfile

```diff
@@ -0,0 +1,12 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+pytest = "*"
+tox = "*"
```

Comment:
Why are we using tox ?
I don't really know it but it seems its doing the same as pipenv.

Reply:
It was useful for testing across multiple Python versions automatically (which `pipenv` cannot not do on its own), but I took it out because we don't *really* need it.

Reply:
Yes, especially if the CI matrix does the same.